main/local_modules/shared_web_theme/static/src/css/base.scss

```diff
@@ -14,4 +14,25 @@ body {
 
 html, body, #wrapwrap {
   height: 0 !important;
-}
\ No newline at end of file
+}
+//.grid-item { width: 25%; }
+
+.grid-item {
+  position: relative;
+  float: left;
+  width: 200px;
+  height: 200px;
+  margin: 5px;
+  padding: 10px;
+  background: #784388;
+  color: #262524;
+}
+.mid-width {
+  width: 50px;
+  height: 50px;
```

Comment:
Color literals like `#888` should only be used in variable declarations; they should be referred to via variable everywhere else.